llvm/include/llvm/MC/MCParser/MCTargetAsmParser.h

```diff
@@ -35,6 +35,7 @@ enum AsmRewriteKind {
   AOK_Align,          // Rewrite align as .align.
   AOK_EVEN,           // Rewrite even as .even.
   AOK_Emit,           // Rewrite _emit as .byte.
+  AOK_CallInput,      // Rewrite in terms of ${N:P}.
```

Comment:
This is an ABI break, if anyone considers backporting to their own LLVM, but I think it's fine in the context of Rust's own use of LLVM.